components/modal/modal.tsx

```diff
@@ -42,7 +42,7 @@ const Modal: FC<ModalProps> = (({ header, height, className, children, subHeader
                         }
                         {
                             isMobile &&
-                            <Leaflet position="fixed" height={height == 'full' ? '80%' : 'fit'} ref={mobileModalRef} show={show} setShow={setShow} title={header} description={subHeader} className={className}>
+                            <Leaflet position="fixed" height={header == "Menu" ? 'full' : "80%"} ref={mobileModalRef} show={show} setShow={setShow} title={header} description={subHeader} className={className}>
```

Comment:
This is not a good abstraction what if we decide to have another full screen modal or change the header of the menu?